js/ui/tag_box.js

```diff
@@ -507,7 +500,7 @@ var TagBox = SelectBox.inherit({
                 .appendTo($tagContent);
 
             $container.append($tagContent);
-        }.bind(this), [this._displayGetterExpr()], this.option("integrationOptions.watchMethod"));
+        }.bind(this), [], this.option("integrationOptions.watchMethod"));
```

Comment:
Wrong usage. See the example here: https://github.com/DevExpress/DevExtreme/blob/8fb7a7f223d0e66c2784fe377cca49809456c76a/js/ui/hierarchical_collection/ui.hierarchical_collection_widget.js#L134